privacyidea/lib/utils.py

```diff
@@ -286,9 +286,17 @@ def modhex_decode(m):
 
 
 def checksum(msg):
+    """
+    Calculate CRC-16 (16-bit ISO 13239 1st complement) checksum.
+    (see Yubikey-Manual - Chapter 6: Implementation details)
+
+    :param msg: input byte string for crc calculation
+    :type msg: bytes
+    :return: crc16 checksum of msg
+    :rtype: int
+    """
```

Comment:
I think it might be a good idea to add an additional test case in ``test_lib_utils.py``. 
Well, we should have had a test case for checksum earlier, but now it looks like we should! ;-)

Reply:
I was going to update the `test_lib_utils.py` and the `lib/utils.py` anyway.